doc/networking.md

```diff
@@ -789,46 +827,50 @@ Remember to enable [IPv4 forwarding](#IPv4-forwarding) for the interfaces.
     admin@example:/config/routing/control-plane-protocol/static/name/default/> leave
     admin@example:/>
 
-> **Note:** You can only have one instance per routing protocol.
 
 ### OSPFv2 Routing
-Infix supports OSPF dynamic routing for IPv4, i.e., OSPFv2.
-Remember to enable [IPv4 forwarding](#IPv4-forwarding) for the
-interfaces you want to run OSPFv2.
+
+The system supports OSPF dynamic routing for IPv4, i.e., OSPFv2.  To
+enable OSPF and set one active interface in area 0:
 
     admin@example:/config/> edit routing control-plane-protocol ospfv2 name default
-    admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/> set ospf area 0.0.0.0 interface e0 enabled true
+    admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/> set ospf area 0.0.0.0 interface e1 enabled
     admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/> leave
     admin@example:/>
 
-> **Note:** You can only have one instance per routing protocol.
+> Remember to enable [IPv4 forwarding](#IPv4-forwarding) for all the
+> interfaces you want to route between.
+
 
 #### OSPF area types
-In addition to *regular* OSPF areas, area types *NSSA* and *Stub* are supported.
 
-To configure a NSSA area with summary routes:
+In addition to *regular* OSPF areas, area types *NSSA* and *Stub* are
+also supported.  To configure an NSSA area with summary routes:
 
     admin@example:/config/> edit routing control-plane-protocol ospfv2 name default
     admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/> set ospf area 0.0.0.1 area-type nssa-area
     admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/> set ospf area 0.0.0.1 summary true
     admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/> leave
     admin@example:/>
 
+
 #### Bidirectional Forwarding Detection (BFD)
-It is possible to enable BFD per interface to speed up detection of
+
+It is possible to enable BFD per OSPF interface to speed up detection of
 link loss.
 
     admin@example:/config/> edit routing control-plane-protocol ospfv2 name default
     admin@example:/config/routing/control-plane-protocol/ospfv2/name/default/ospf/> set area 0.0.0.0 interface e0 bfd enabled true
```

Comment:
you change this to e1 above, maybe search and replace it?

Reply:
Overzealous change (we don't have e0 anymore), but I'm reverting this because changing all other instances would become a bigger headache.  Thank you for catching this, Eagle Eye! :eagle: